<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpAppearancePage</class>
 <widget class="QWidget" name="HelpAppearancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>547</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure help viewer appearance&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="standardFontButton">
        <property name="toolTip">
         <string>Press to select the standard font</string>
        </property>
        <property name="text">
         <string>Standard Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="standardFontSample">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Times 16</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="fixedFontButton">
        <property name="toolTip">
         <string>Press to select the fixed-width font</string>
        </property>
        <property name="text">
         <string>Fixed-Width Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fixedFontSample">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Courier 13</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Colours</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Background colour of secure URLs:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="secureURLsColourButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for secure URLs.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Images</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="autoLoadImagesCheckBox">
        <property name="toolTip">
         <string>Select to load images</string>
        </property>
        <property name="text">
         <string>Load images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Style Sheet</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User Style Sheet:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="E5PathPicker" name="styleSheetPicker" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the file name of a user style sheet</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tabsGroupBox">
     <property name="title">
      <string>Tabs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="tabsCloseButtonCheckBox">
        <property name="text">
         <string>Show only one close button instead of one for each tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="warnOnMultipleCloseCheckBox">
        <property name="toolTip">
         <string>Select to issue a warning, if multiple tabs are about to be closed</string>
        </property>
        <property name="text">
         <string>Warn, if multiple tabs are about to be closed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>479</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>standardFontButton</tabstop>
  <tabstop>fixedFontButton</tabstop>
  <tabstop>secureURLsColourButton</tabstop>
  <tabstop>autoLoadImagesCheckBox</tabstop>
  <tabstop>styleSheetPicker</tabstop>
  <tabstop>tabsCloseButtonCheckBox</tabstop>
  <tabstop>warnOnMultipleCloseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
